.circleci/config.yml

```diff
@@ -121,13 +121,19 @@ jobs:
     <<: *defaults
     machine:
       image: ubuntu-1604:202007-01
+    parameters:
+      k8s-version:
+        type: string
     steps:
       - <<: *should_test
       - checkout
       - <<: *exports
+      - run: |
+          echo "export MINIKUBE_KUBERNETES_VERSION=<< parameters.k8s-version >>" >> $BASH_ENV
       - attach_workspace:
           at: /tmp/go
       - <<: *restore_workspace
+      - run: sudo apt-get update && sudo apt-get install -y conntrack
```

Comment:
what's the purpose of `conntrack`?

Reply:
The job for [minikube_kafka-v1.18.9](https://app.circleci.com/pipelines/github/kubeless/kafka-trigger/49/workflows/90744719-a105-4f1d-a23b-923285a934be/jobs/306)  fails if `conntrack` is not installed.

> Kubernetes 1.18.9 requires conntrack to be installed in root's path

Not sure why is required though.